src/styles/RegisterStyles.js

```diff
@@ -0,0 +1,36 @@
+const styles = theme => ({
```

Comment:
Change to FormStyles. This will be the same styling used for both forms, after all!